introduction/getting-started.md

```diff
@@ -54,27 +52,24 @@ corresponding version number. There are many ready-to-use
 many as you like.
 
 Last but not least, there's a very special [admin
-qube](/doc/glossary/#admin-qube) which, as the name suggests, is used to
-administer your entire system. There's only one admin qube, and it's called
-[dom0](/doc/glossary/#dom0). You can think of it as the master qube, holding
-ultimate power over everything that happens in Qubes OS. Dom0 is more trusted
-than any other qube. If dom0 were ever compromised, it would be "game over."
-The entire system would effectively be compromised. That's why everything in
-Qubes OS is specifically designed to protect dom0 and ensure that doesn't
+qube](/doc/glossary/#admin-qube) used to administer your entire system. 
+There's only one admin qube, and it's called [dom0](/doc/glossary/#dom0). 
+You can think of it as the master qube, holding ultimate power over everything that happens in Qubes OS. 
+Dom0 is the most trusted one of all qubes. If dom0 were ever to be compromised, it would be "game over"- an effective compromise of the entire system.
+That's why everything in Qubes OS is specifically designed to protect dom0 and ensure that doesn't
 happen. Due to its overarching importance, dom0 has no network connectivity and
 is used only for running the [desktop
 environment](https://en.wikipedia.org/wiki/Desktop_environment) and [window
 manager](https://en.wikipedia.org/wiki/Window_manager). Dom0 should never be
 used for anything else. In particular, you should never run user applications
-in dom0. (That's what your app qubes are for!)
+in dom0. (That's what your app qubes are for!) In short, do not interact directly with dom0 in any way, shape or form.
```

Comment:
I think this is too harsh; people should after all update their dom0, and even though it does happen magically with a tool, it does involve interacting with dom0.